docs/source/development/traffic_ops_api/v12/cachegroup_fallbacks.rst

```diff
@@ -0,0 +1,209 @@
+
+  +-----------------+---------------------------------------------------------------------------+
+  | Name            | Description                                                               |
+  +=================+===========================================================================+
+  | cacheGroupId    | The id of the cache group whose backup configurations has to be retrieved |
+  +-----------------+---------------------------------------------------------------------------+
+  | fallbackId      | The id of the fallback cache group associated with a cache group          |
+  +-----------------+---------------------------------------------------------------------------+
+
+  **Response Properties**
+
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | Parameter                         | Type   | Description                                                              |
+  +===================================+========+==========================================================================+
+  |                                   | array  | parameters array                                                         |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>cacheGroupId``                 | int    | Cache group id                                                           |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>fallbackId``                   | int    | fallback cache group id                                                  |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>cacheGroupName``               | string | Cache group name                                                         |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>fallbackName``                 | string | Fallback cache group  name                                               |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>fallbackOrder``                | int    | Ordering list in the list of backups                                     |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+
+  **Response Example** ::
+
+    {
+       "response": [
+          {
+             "cacheGroupId":1,
+             "cacheGroupName":"GROUP1",
+             "fallbackId":2,
+             "fallbackOrder":10,
+             "fallbackName":"GROUP2"
+          }
+       ]
+    }
+
+|
+
+**POST /api/1.2/cachegroup_fallbacks**
+
+  Creates/ Updates fallback list for the cache group.
```

Comment:
Can you add a little more detail here. How does this work exactly? Send in an array to the POST. for each item in the array, if doesn't exist, create it. if exists, update it?